src/ORM/Association/BelongsToMany.php

```diff
@@ -301,7 +302,7 @@ protected function _joinCondition($options)
      * @param \Cake\ORM\Query $fetchQuery The query to get results from
      * @param array $options The options passed to the eager loader
      * @return array
-     * @throws \RuntimeException when the association property is not part of the results set.
+     * @throws RuntimeException when the association property is not part of the results set.
```

Comment:
Yes doc blocks should have absolute class references. The API generator does not have a problem but some IDEs take it as relative path.